app/javascript/router/Landing.vue

```diff
@@ -1,18 +1,45 @@
 <template>
-  <div>
-    <p>Route: {{ routeName }}</p>
+  <div class="m-12">
+    <div class="bg-gray-200 p-10 my-4 rounded-md">
+      <p class="text-5xl font-bold">BitSplit</p>
+      <p
+        class="font-light"
+      >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam</p>
+    </div>
+    <div
+      class="flex flex-col justify-center md:flex-row md:justify-start bg-gray-200 p-10 rounded-md"
+    >
+      <div class="flex flex-col md:flex-row md:items-start text-center px-4 py-2">
+        <LinkButton
+          classmod="bg-blue-500 hover:bg-blue-700 mx-4 my-3 md:my-0"
+          :fieldDisabled="false"
+          fieldPlaceholder="Iniciar sesion"
+          route="sign-in"
+        />
+        <LinkButton
+          classmod="bg-blue-500 hover:bg-blue-700 mx-4"
+          :fieldDisabled="false"
+          fieldPlaceholder="Registrarse"
+          route="sign-up"
+        />
```

Comment:
Haciendo eso queda mas legible el código
```suggestion
        <LinkButton classmod="my-3 md:my-0" :disabled="false"route="sign-in">
          Iniciar sesion
        </LinkButton>
        <LinkButton :disabled="false" route="sign-up">
          Registrarse
        </Button
```